Gemfile

```diff
@@ -5,7 +5,7 @@ gemspec
 gem 'aamva', github: '18F/identity-aamva-api-client-gem', tag: 'v3.4.1'
 gem 'identity-doc-auth', github: '18F/identity-doc-auth', tag: 'v0.3.1'
 gem 'lexisnexis', github: '18F/identity-lexisnexis-api-client-gem', tag: 'v2.4.1'
-gem 'proofer', github: '18F/identity-proofer-gem', tag: 'v2.7.0'
+gem 'proofer', git: 'https://github.com/18F/identity-proofer-gem.git', ref: 'd64adb3'
```

Comment:
Is the plan to wait for the proofer version to be merged and push a new tag there, so it can be updated here?

Reply:
yes